packages/manager/apps/sign-up/src/index.html

```diff
@@ -21,12 +21,12 @@
                     data-ng-href="{{ $ctrl.logoUrl }}"
                 >
                     <img
-                        src="assets/img/logo-ovh-baseline.png"
+                        src="assets/img/Logo OVHcloudTM.png"
```

Comment:
This logo should be displayed for a single region.

Could you please keep the file naming convention? (no space, lower case and `-`).

Reply:
Extra request: Do you think that the `<title>` of the page could be also updated from `OVH` to `OVHcloud`?

Reply:
@antleblanc yes, done :)